docs/jwst/stpipe/config_asdf.rst

```diff
@@ -3,7 +3,11 @@
 ASDF Configuration Files
```

Comment:
Not sure we should continue to call them "configuration" files, because of the historical connection between that term and an actual .cfg file. I know we're using the word "configuration" in a generic sense here, but to avoid possible confusion, we should perhaps always now refer to them as "parameter" files.